lib/udf_findhotel_utils.rb

```diff
@@ -446,25 +446,19 @@ def map_device(device):
                         "Desktop": "d",
                         "Tablet": "t"}.get(device)
 
-            def destination_lcase(des):
-                if des != None:
-                 return(des.lower())
-                else:
-                  return('')
-
             try:
                 # items come either from query itself of from label within it
                 items = get_query_items(url)
                 label_items = get_label_items(items)
                 items.update(label_items)
 
                 key = {
-                    "campaign_id": get_value(items, "camp").lower(),
-                    "ad_group_id": get_value(items, "adgrp").lower(),
-                    "ad_id": get_value(items, "ad").lower(),
-                    "device": map_device(get_value(items, "dev")),
-                    "hotel_id": hotel_id or get_value(items, "hotelID"),
-                    "place_name": get_value(items, "des").lower()}
+                  "campaign_id": get_value(items, "camp").lower(),
+                  "ad_group_id": get_value(items, "adgrp").lower(),
```

Comment:
why not dealing with nulls in the same way also for campaign_id, ad_group_id etc?